test/query/shorthand.test.ts

```diff
@@ -297,34 +427,19 @@ describe('query/shorthand', () => {
       assert.equal(str, 'a,q,sort={"field":"a","op":"mean","order":"descending"}');
     });
 
-    it('should return correct fieldDefShorthand string for bin with sort field definition object', () => {
```

Comment:
The old example was incorrect and its only purpose was to demonstrate bin with a nested parent property. Since we already have bin and scale test I thought it was unnecessary and removed it.